lib/fee_threshold.rb

```diff
@@ -0,0 +1,42 @@
+class FeeThreshold
```

Comment:
I think this is really good preemptive extraction for the refactoring we're doing :+1: